src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator start work");
+    }
+
+    public void stopWork() {
+        System.out.println("Excavator stop work");

```

Comment:
Similarly, the stop message 'Excavator stop work' is not very informative. Consider rephrasing it to 'Excavator has stopped working.' to clearly communicate the action.